src/CSET/operators/constraints.py

```diff
@@ -106,7 +107,10 @@ def no_levels(cube):
 
     # Filter the coordinate to the desired levels.
     # Dictionary unpacking is used to provide programmatic keyword arguments.
-    return iris.Constraint(**{coordinate: levels})
+    if levels == "*":
+        return iris.Constraint(**{coordinate: lambda cell: True})
```

Comment:
This is mostly coincidentally working currently, as strings *are* Iterables. Its a bit unintuitive however, so it would be easier to follow if this if check was on line 95, and then only ran the rest of the code if not triggered.